packages/directory/addon/templates/components/dir-new-button.hbs

```diff
@@ -0,0 +1,12 @@
+{{!-- Copyright 2018, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+{{#basic-dropdown as |dd|}}
+  {{#dd.trigger classNames='dir-new-button__trigger btn-blue btn-blue-primary'}}New{{/dd.trigger}}
+  {{#dd.content class='dir-new-button__content'}}
+    {{#each documentTypes as |type|}}
+      {{#link-to (concat (get type 'name') '.new') class='dir-new-button__dropdown-option' tagName='div'}}
```

Comment:
Should we have the `route` in the json array? there could be a case where the type and the route name could be different.